central/main.go

```diff
@@ -373,6 +374,7 @@ func servicesToRegister(registry authproviders.Registry, authzTraceSink observe.
 		probeUploadService.Singleton(),
 		processIndicatorService.Singleton(),
 		processBaselineService.Singleton(),
+		processListeningOnPorts.Singleton(),
```

Comment:
You should move adding the service singleton to the condition at l.395, i.e. only add the service if postgres is enabled.

Reply:
Done